Chirp/ViewController.swift

```diff
@@ -7,13 +7,60 @@
 
 import UIKit
 
-class ViewController: UIViewController {
+class ViewController: UITabBarController {
 
+
+    
     override func viewDidLoad() {
         super.viewDidLoad()
         // Do any additional setup after loading the view.
+  
+        
+        let vc1 = FirstViewVC()
+        let vc2 = SecondViewVC()
+        let vc3 = ThirdViewVC()
+        self.setViewControllers([vc1, vc2, vc3], animated: true)
+        
+        guard let items = self.tabBar.items else {
+            return
+        }
+        
+        let tabIconImages = ["ellipsis.message.fill", "person.2.fill", "squareshape.split.2x2"]
```

Comment:
Can we create an extension of UIImage and add create the icons there instead?
If you pull the latest version of development and rebase, you'll notice that there's an `Extensions` folder. Inside this folder, create a new file `UIImage+Extension` and add these icons there.
E.g
```swift
extension UIImage {
    static let ellipsisMessageFill = UIImage(systemName: "ellipsis.message.fill")!
    static let personFill = UIImage(systemName: "person.2.fill")!
    ...
}
```